test_conformance/extensions/cl_khr_command_buffer/command_buffer_test_fill.cpp

```diff
@@ -47,14 +47,30 @@ struct FillImageKHR : public BasicCommandBufferTest
                                           nullptr, nullptr);
         test_error(error, "clEnqueueCommandBufferKHR failed");
 
-        std::vector<cl_char> output_data(data_size);
+        std::vector<cl_char> output_data_1(data_size);
         error = clEnqueueReadImage(queue, image, CL_TRUE, origin, region, 0, 0,
-                                   output_data.data(), 0, nullptr, nullptr);
+                                   output_data_1.data(), 0, nullptr, nullptr);
 
         for (size_t i = 0; i < data_size; i++)
         {
             CHECK_VERIFICATION_ERROR(static_cast<cl_char>(pattern),
-                                     output_data[i], i);
+                                     output_data_1[i], i);
+        }
+
+        /* Check second enqueue of command buffer */
```

Comment:
Can we reinitalize  `image` before the second enqueue, e.g fill it with a different color